ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml

```diff
@@ -79,7 +79,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: api.ci
-  cron: 0 */6 * * *
+  cron: 0 */12 * * *
```

Comment:
Not sure if this make sense, but wouldn't it be better to choose a slightly different timeslot for each run, in order to stretch a bit the load on the infrastructure ? (if its running with elastic resources anyway, then of course this doesn't matter).

Reply:
I'd say these few jobs will not make a difference among the large pool of tests from the whole OpenShift team.